runelite-api/src/main/java/net/runelite/api/events/DraggingWidgetChanged.java

```diff
@@ -26,6 +26,9 @@
 
 import lombok.Data;
 
+/**
+ * This event is used in the Prayer Reorder Plugin to determine when a prayer is moved.
```

Comment:
I don't know if mentioning plugins here is ideal. I would rather know what it actually does instead of first having to look at the prayer plugin to find out what it does.